Public/Src/Utilities/Configuration/IRootModuleConfiguration.cs

```diff
@@ -23,5 +23,11 @@ public interface IRootModuleConfiguration : IModuleConfiguration
         /// </summary>
         [CanBeNull]
         IReadOnlyList<RelativePath> SearchPathEnumerationTools { get; }
+
+        /// <summary>
+        /// List of path fragments of tools using its own incrementality
+        /// </summary>
+        [CanBeNull]
+        IReadOnlyList<RelativePath> IncrementalTools { get; }

```

Comment:
Curious why this is on the root configuration. Should we do it per pip? Users need to set a bit on the pip anyway...

Reply:
It is very similar to searchpathenumeration tools. It is global.